src/main/java/bio/terra/workspace/service/iam/SamUtils.java

```diff
@@ -1,4 +1,4 @@
-package bio.terra.workspace.common.utils;
+package bio.terra.workspace.service.iam;
```

Comment:
Agreed that utils is a weird name for a bunch of constants. I think this makes sense as a model since other parts of the code call into SamService with those values (e.g. calling `samService.isAuthorized` with a resource type), though I'm not 100% on the name.